### PR TITLE
Update Commandline.md

### PR DESCRIPTION
**Pull request by BrownSlaughter**
## Description
Updated commandline.md
## Motivation and Context

needed update as outdated

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by BrownSlaughter** [comment]
There is actually less tabs than the original, not sure why it shifted over so far, let me remove some

**Comment by onilton** [comment]
Please, don't replace space with tabs.

**Comment by BrownSlaughter** [comment]
yeah that last commit was before I realised the issue, still being sorted now

File: docs/extras/commandline.md [hunk 1]
```diff
@@ -128,110 +149,140 @@
...
                             Use spawnpoint scanning (instead of hex grid). Scans
-                            in a circle based on step_limit when on DB [env var:
+                            in a circle based on step_limit when on DB. [env var:
                             POGOMAP_SPAWNPOINT_SCANNING]
-      -speed [SPEED_SCANNING], Use hex scanning with 5 passes per hour to identify
-                            spawns, and then transition to spawn scanning based on
-                            closest spawn to the worker.
-      -kph                  Set speed limit in kilometers/hour. Default is 35 kp/h
-                            For use with -speed scanning.
-      -bh                   Use beehive with -wph workers per hive until
```
Comment (onilton) [inline docs/extras/commandline.md]: Where did -bh and -wph went to?

Comment (BrownSlaughter) [inline docs/extras/commandline.md]: Strange, must have been cut out while deleting tabs
I replaced them

**Comment by onilton** [comment]
@tomballgithub Actually remember that --ss accepts  an argument that can be a file name, usually something like filename.json

**Comment by tomballgithub** [comment]
@onilton Correct, thank you for clarifying, haven't used that mode in a while.

**Pull request merged**